cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/table/field/FieldType.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data.table.field;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * Contains the name and type information of a {@link Field}.
+ */
+public final class FieldType {
+  /**
+   * Supported data types.
+   */
+  public enum Type {
+    INTEGER,
+    LONG,
+    FLOAT,
+    DOUBLE,
+    STRING
+  }
+
+  static final Set<Type> PRIMARY_KEY_TYPES = ImmutableSet.of(Type.INTEGER, Type.LONG, Type.STRING);
```

Comment:
Why define constant in this class while it is only used in another class `Fields`?